config.py

```diff
@@ -13,7 +13,7 @@
     for name, num in vars(socket).items()
     if name.startswith("IPPROTO")
 }
-real_machine_ip_address = socket.gethostbyname(socket.gethostname())
+real_machine_ip_address = socket.gethostbyname('')
```

Comment:
why is this changed?

Reply:
@dhirensr Sorry accidentally committed that code. It was giving some error like no host name present. So I fixed it in local. Removed it.